tests/test_convert.py

```diff
@@ -60,6 +62,28 @@ def test_config():
         ) == sorted(config_preset.keys())
 
 
+def test_build_path(fx_tempdir: str):
```

Comment:
Do you mean `expand_path`? Oh, I see, expand_path is used by build_path. Looks good!

Reply:
Thanks - this is a good point, I think these tests would be better as `expand_path` specific tests and will make an update to address this.